### PR TITLE
Wip/cragw/k2 19x

### PR DESCRIPTION
**Pull request by CragW**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupd/fwupd-enums.h [hunk 1]
```diff
@@ -556,6 +556,14 @@ typedef enum {
  * Since: 1.9.3
  */
 #define FWUPD_DEVICE_FLAG_ONLY_EXPLICIT_UPDATES (1ull << 51)
+/**
```
Comment (CragW) [inline libfwupd/fwupd-enums.h:559]: Should we pull 52 to 1.9.x too? It is being skipped for now.

```
FWUPD_DEVICE_FLAG_CAN_EMULATION_TAG = 1ull << 52,
```

File: libfwupd/fwupd-enums.h [hunk 2]
```diff
@@ -556,6 +556,14 @@ typedef enum {
...
+ * FWUPD_DEVICE_FLAG_INSTALL_SKIP_VERSION_CHECK:
+ *
+ * The device doesn't require verification of the newly installed version.
+ *
+ * Since: 1.9.26
```
Comment (superm1) [inline libfwupd/fwupd-enums.h:564]: 1.9.27 now

Comment (CragW) [reply libfwupd/fwupd-enums.h]: Fixed, thank you!

File: libfwupd/fwupd-enums.h (hunk 1 above)
Comment (CragW) [reply libfwupd/fwupd-enums.h:559]: Understand, let's skip this one.

**Review by superm1 — APPROVED** [review]

**Pull request merged**